data.sql

```diff
@@ -12,4 +12,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Charmander', '2020-02-08', 0, false, -11.00), ('Plantmon', '2021-11-15', 2, true, -5.70), ('Squirtle', '1993-04-02', 3, false, -12.13),
     ('Angemon', '2005-06-12', 1, true, -45.00), ('Boarmon', '2005-06-07', 7, true, 20.40), ('Blossom', '1998-10-13', 3, true, 17.00),
     ('Ditto', '2022-05-14', 4, true, 22.00)
-;
\ No newline at end of file
+;
+
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE species.name = 'Digimon') WHERE animals.name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE species.name = 'Pokemon') WHERE animals.name NOT LIKE '%mon';
+UPDATE animals SET owners_id = (SELECT id FROM owners WHERE owners.full_name = 'Sam Smith') WHERE animals.name = 'Agumon';
+UPDATE animals SET owners_id = (SELECT id FROM owners WHERE owners.full_name = 'Jennifer Orwell') WHERE animals.name IN ('Gabumon', 'Pikachu');
+UPDATE animals SET owners_id = (SELECT id FROM owners WHERE owners.full_name = 'Bob') WHERE animals.name IN ('Plantmon', 'Devimon');
+UPDATE animals SET owners_id = (SELECT id FROM owners WHERE owners.full_name = 'Melody Pond') WHERE animals.name IN ('Charmander', 'Blossom', 'Squirtle');
+UPDATE animals SET owners_id = (SELECT id FROM owners WHERE owners.full_name = 'Dean Winchester') WHERE animals.name IN ('Angemon', 'Boarmon');
```

Comment:
- [ ] Similarly as mentioned in the review summary, kindly include the following highlighted sections inside your data.sql file. You might have erroneously omitted them.

![image](https://user-images.githubusercontent.com/86318284/215243344-611c83c5-7e83-4f42-95e1-8b1bb10307ef.png)